### DISCUSSION TITLE
Could not create share link Unable to get public url

### DISCUSSION THREAD

**Discussion by raahulgupta07** [lllyasviel/Fooocus#2818] [Q&A] [answered]
Could not create share link. Missing file: /usr/local/lib/python3.10/dist-packages/gradio/frpc_linux_amd64_v0.2. 

Please check your internet connection. This can happen if your antivirus software blocks the download of this file. You can install manually by following these steps: 
![Uploading Screenshot 2024-04-27 at 1.26.50 PM.png…]()

**Comment by poor7** [comment, marked as answer]
Today the Gradio Share API crashes periodically. Because of this, an error is displayed and the link is not created.
https://status.gradio.app/793595965

**Reply by puntomaupunto** [reply]
thank you!

**Reply by Achex** [reply]
The status is saying operational but the share link is still not getting generated for me. Is anyone getting the same?

**Reply by ILikeToasters** [reply]
It has been saying that off and on. I am not sure they have fully dealt with whatever the issue is. At least that is how it appears to me.

**Reply by mashb1t** [reply]
As of my most recent check 10 minutes ago it seems to be working again.

**Reply by Achex** [reply]
working again for me

**Comment by puntomaupunto** [comment]
diese Morgen war es kaputt, aber jetzt es funktioniert wieder.
ciao, .mau.
